examples/tfengine/generated/devops/cicd/configs/tf-apply.yaml

```diff
@@ -22,43 +22,11 @@ steps:
     args: ["version"]
     id: Terraform version
 
-  # live/cicd
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"
```

Comment:
use the `cft` variable for container as done in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/477/files

Reply:
same for the other ones